pages/deploy/deploy_aws.md

```diff
@@ -563,6 +567,15 @@ disable any other core feature.
 | **Feature**           | **Module**     | **Description**                                                                  |   
 |-----------------------|----------------|----------------------------------------------------------------------------------|
 | env_aws_actions       | environments   | Get AWS Credentials and assume Environment Group IAM roles from data.all's UI    |
+
+### Add Customization
+
+At places, you can use few customizations to create a tailored data.all for your needs. Below are few customizations available,
```

Comment:
Can re-word the following and move in between "Disable module features" and "Disable core features" to...

header -->  "Customizing Module Features"

In addition to disabling / enabling, some module features allow for some additional customization to create a tailored data.all for your needs. Below is an example of how one could customize module features in the `config.json` and a list of all supported customizations:

```json
    ...
    "datasets": {
        "features": {
            "custom_confidentiality_mapping": {
                 "Public" : "Unclassified",
                 "Private" : "Official", 
                 "Confidential" : "Secret",
                 "Very Highly Confidential" : "Secret"
             }
        }
    }
    ...
```

TABLE HERE (do need the example json in the description if have above example)

Reply:
Sure Let me do that and update the PR